custom/icds_reports/tests/reports/test_new_born_with_low_weight.py

```diff
@@ -265,7 +264,7 @@ def test_chart_data(self):
                     },
                     {
                         "loc_name": "st1",
-                        "percent": 100.0
+                        "percent": 66.66666666666667
```

Comment:
does it need that much precision?

Reply:
Probably not. It's possible that the UI fixes that. @lwyszomi any reason these tests have this amount of precision?

Reply:
@emord @snopoke we changing format on the UI to two decimal places.